contents/scummvm.oscmeta

```diff
@@ -0,0 +1,23 @@
+{
+    "information": {
+        "name": "ScummVM",
+        "author": "The ScummVM Team",
+        "category": "emulators",
+        "peripherals": ["Wii Remote", "GameCube Controller", "SDHC"],
+        "version": "2.7.0"
+    },
+    "source": {
+        "type": "manual"
+    },
+	"treatments": [
+		{
+			"treatment": "web.download", "arguments": ["https://downloads.scummvm.org/frs/scummvm/2.7.0/scummvm-2.7.0-wii.zip", "scummvm-2.7.0-wii.zip"]
+		},
+        {
+            "treatment": "archive.extract", "arguments": ["scummvm-2.7.0-wii.zip", "apps/ScummVM/"]
+        },
```

Comment:
wrong path

Reply:
I'll check the archive with the one from osc.

Reply:
That's the same thing... https://i.ibb.co/8NstnBw/screenshot.png

Reply:
case sensitivity.